src/go/rpk/pkg/config/rpk_yaml.go

```diff
@@ -329,6 +330,23 @@ func (y *RpkYaml) WriteAt(fs afero.Fs, path string) error {
 	return rpkos.ReplaceFile(fs, path, b, 0o644)
 }
 
+func (r *RpkCloudCluster) IsServerless() bool {
+	return r != nil && r.Type == publicapi.ServerlessClusterType
+}
+
+func (r *RpkCloudCluster) CheckClusterURL() (string, error) {
+	if r == nil {
+		return "", fmt.Errorf("cluster information not present in current profile. Please delete and re-create the current profile with 'rpk profile create --from-cloud'")
```

Comment:
`. ` => `; `

Reply:
Fixed in https://github.com/redpanda-data/redpanda/pull/17023/commits/8187221597016ec3950c5fe72030ed03eab7d022